### ISSUE TITLE
Add alt attribute to the EU logo

### ISSUE THREAD

**Issue by andreskytt** [okestonia/jkan#398]
1.1 Text Alternatives: Provide text alternatives for any non-text content

Success Criteria 1.1.1 Non-text Content (A)

Check 1: img element missing alt attribute.

`Line 207, Column 5:<img src="/img/erf_logo.jpg" width="150"/>`
